### PR TITLE
ci: Setup a semi-automated release process (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
[ENG-47](https://linear.app/n8n/issue/ENG-47)
[ENG-60](https://linear.app/n8n/issue/ENG-60)

### CODE REVIEW

File: .github/workflows/release-publish.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          node-version: 16.x
+          cache: 'pnpm'
+      - run: pnpm install --frozen-lockfile
+
+      - name: Build
+        run: pnpm build
+
+      - name: Publish to NPM
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
```
Comment (netroy) [inline .github/workflows/release-publish.yml:39]: One of the admins need to add `NPM_TOKEN` to [our repo secrets](https://github.com/n8n-io/n8n/settings/secrets/actions).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -195,6 +196,22 @@ If that gets executed in one of the package folders it will only run the tests
 of this package. If it gets executed in the n8n-root folder it will run all
 tests of all packages.
 
+## Releasing
+
+To start a release, trigger [this workflow](https://github.com/n8n-io/n8n/actions/workflows/release-create-pr.yml) with the SemVer release type, and select a branch to cut this release from. This workflow will then
+
+1. Bump versions of packages that have changed or have depndencies that have changed
```
Comment (janober) [inline CONTRIBUTING.md:203]: typo in "depndencies"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -195,6 +196,22 @@ If that gets executed in one of the package folders it will only run the tests
...
+2. Update the Changelog
+3. Create a new branch called `release/${VERSION}`, and
+4. Create a new pull-request to track any further changes that need to be included in this release
+
+Once ready to release, simply merge the pull-request.
+This triggers [another workflow](ttps://github.com/n8n-io/n8n/actions/workflows/release-publish.yml), that will
```
Comment (janober) [inline CONTRIBUTING.md:209]: Looks like the `h` in `https` is missing

Comment (janober) [inline CONTRIBUTING.md:209]: Looks like the `h` in `https` is missing

Comment (janober) [inline CONTRIBUTING.md:209]: Looks like the `h` in `https` is missing

**Comment by janober** [comment]
Got released with `n8n@0.217.1`

**Pull request merged**